Cargo.toml

```diff
@@ -29,5 +29,5 @@ path = "src/lib.rs"
 crate-type = [ "dylib" ]
 plugin = true
 
-[dev_dependencies.gfx]
-git = "https://github.com/gfx-rs/gfx-rs"
+[dev_dependencies]
+gfx = "*"
```

Comment:
If you have your `.cargo/config` set up it should not be a problem. Here is mine:

~~~toml
paths = [ "gfx-rs/", "gfx_gl/", "gfx_macros/", "gfx_device_gl/", "gfx_examples/" ]
~~~